spec/models/task_spec.rb

```diff
@@ -9,4 +9,12 @@
       expect(task).to_not be_valid
     end
   end
+
+  context 'with completed attributes when creating' do
+    let(:task) { build :task, completed: true}
+
+    it 'is invalid' do
+      expect(task).to_not be_valid
```

Comment:
I'm wondering if there is another way to be more specific in testing the validity of this model? How can you be sure that it is invalid because of the completed attribute on creation?

Reply:
Good question. I added another test when set task's completed back to false, it should be valid.

Reply:
Also to answer your first question, if I had more time, I would implement the user story "A user should have access to only their tasks. (may require devise or a creative, lightweight auth system)". Because I haven't used devise. I think it must be very interesting.